### ISSUE TITLE
CORS Error

### ISSUE THREAD

**Issue by ankitvats1** [totaljs/flow#72]
Hi @petersirka,

I'm trying to hit **flow API** from outside the flow, getting the CORS error -

`_Access to XMLHttpRequest at 'http://localhost:8111/api/' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource._`

Is there any way to update the CORS setting?

Basically, I just want to create a new **flowstream** using the API.
`axios.post("http://localhost:8111/api/", {schema: "streams_save", data{}})`

**Comment by petersirka** [comment]
Hi @ankitvats1,
you must use **REST CORS component** (import it or download directly): https://github.com/totaljs/flowstreamcomponents/blob/main/components/restcors.html
